WordPress/gutenberg
# Block Parser: Disable console.info in production

## Description

The block validation will display a console.info for blocks succesfully updated during the validation process. This PR wraps that console.info in a check for production so it is not displayed for produciton instances.

Fixes #33148

## How has this been tested?

- Open a post created in a previous version (sample post with buttons works)

- Confirm validation notice is shown in console (info needs to be checked)
<img src="https://user-images.githubusercontent.com/45363/124192482-3382af80-da7a-11eb-812b-fefa2c87c958.png" width="401">

- Apply fix and run against production build
    - Production build created using: `npm run build`
   
- Test again and confirm no console.info

## Types of changes

Wraps console.info with a production environment check.
